src/main/java/hillel/spring/repository/HelloRepository.java

```diff
@@ -0,0 +1,19 @@
+package hillel.spring.repository;
+
+import hillel.spring.database.Database;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class HelloRepository {
+
+    private final Database database;
+
+    public HelloRepository() {
+        database = new Database();
+
+    }
+
+    public Database getDatabase() {
```

Comment:
Database shouldn't be published. Repository should provide public methods for searching/creating items, and nobody should know where items are stored (in real DB or in custom array-based implementation). It is an Encapsulation